### DISCUSSION TITLE
Add sort by: Not Completed

### DISCUSSION THREAD

**Discussion by bauera-cloud** [codewars/codewars.com#3105] [Ideas]
It would be convenient to see the kata that I didn't complete.

**Comment by hobovsky** [comment]
Doensn't this work for you?

![image](https://github.com/codewars/codewars.com/assets/23709795/5bfae47e-f660-42e4-acd5-7db64a6992fa)

**Comment by bauera-cloud** [comment]
Oh I didn't notice that. I just thought it would be in the sort by category.

Thank you.

**Reply by hobovsky** [reply]
Closing as resolved then.
